.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Rapier CI build
+
+on:
+  workflow_run:
+    workflows: [ci, links]
+    branches: [master]
+    types: [completed]
+  workflow_dispatch:
+    # workflow dispatch is to manually trigger the workflow,
+    # useful if we want to bring an older version online or an upload failed somehow.
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check Rsync version
+        run: rsync --version
```

Comment:
we may have to manually install rsync, I'm not sure it's included in ubuntu-latest

```
      - name: Install rsync
        run: sudo apt-get update && sudo apt-get -y install rsync
```